pkg/controllers/securitypolicy/securitypolicy_controller.go

```diff
@@ -137,12 +139,16 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		obj = &v1alpha1.SecurityPolicy{}
 	}
 
-	log.Info("reconciling SecurityPolicy CR", "securitypolicy", req.NamespacedName)
+	log.Info("Reconciling SecurityPolicy CR", "securitypolicy", req.NamespacedName)
+	startTime := time.Now()
+	defer func() {
+		log.Info("Finished reconciling SecurityPolicy CR", "securitypolicy", req.NamespacedName, "duration(ms)", time.Since(startTime).Milliseconds())
+	}()
+
 	metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerSyncTotal, MetricResType)
 
 	if err := r.Client.Get(ctx, req.NamespacedName, obj); err != nil {
-		// IgnoreNotFound returns nil on NotFound errors.
-		if client.IgnoreNotFound(err) == nil {
+		if apierrors.IsNotFound(err) {
```

Comment:
@TaoZou1 please help confirm this is correct change in the code.

Reply:
correct

Reply:
Thanks for confirmation.